comms/comms.go

```diff
@@ -78,3 +83,14 @@ func NewToDDComms(cfg config.Config) (*toddComms, error) { // TODO: Return Packa
 	return &tc, nil
 
 }
+
+// NewAgentComms returns a comms instance configured for agent usages.
+//
+// TODO: accept an interface for cache instead of concrete type
+func NewAgentComms(cfg config.Config, ac *cache.AgentCache) (*toddComms, error) {
```

Comment:
I noticed you changed the reference to `NewToDDComms` to point here instead (in https://github.com/toddproject/todd/pull/126/files#diff-8a0edf5a42325103a551ddd3ac9616c8R79) but left the old `NewToDDComms` function.

Is the intention to eventually retire `NewToDDComms`, but just didn't want to bloat this PR? If so, a comment indicating so would be appreciated